SPECS/e2fsprogs/e2fsprogs.spec

```diff
@@ -62,7 +62,17 @@ rm -rf %{buildroot}%{_infodir}
 %find_lang %{name}
 
 %check
-make %{?_smp_mflags} check
+# Mult-threaded runs are flaky.
+make -j1 check
```

Comment:
As far as I know, our package test scripts use the last exit code output in the `%check` section to determine whether tests failed or not. Could we save the exit code of the call to `make`, and use that to ensure the proper code is returned? My concern is that having the log output here will mask the actual exit code.

Reply:
Ah, right, good catch. Let me fix that.